src/compiler/compiler.rs

```diff
@@ -1399,6 +1453,14 @@ mod test {
         let creator = new_creator();
         let runtime = single_threaded_runtime();
         let pool = runtime.handle();
+        next_command(
+            &creator,
+            Ok(MockChild::new(
+                exit_status(1),
+                "",
+                "gcc: error: unrecognized command-line option '-vV'; did you mean '-v'?",
+            )),
+        );
```

Comment:
Iirc, the result will be cached, so the impact of another resolution call is very small, even for small projects. Correct me if I am wrong.

Reply:
Is this comment to https://github.com/mozilla/sccache/pull/1897#discussion_r1466719055?

Reply:
This is a principled question, if we incur much overhead for the compiler resolution with this changeset.

Reply:
I am not sure, to be honest. But if the results are cached then it `should` be ok.